skyvern/webeye/scraper/domUtils.js

```diff
@@ -1134,11 +1134,7 @@ function buildElementObject(frame, element, interactable, purgeable = false) {
   }
 
   if (elementTagNameLower === "input" || elementTagNameLower === "textarea") {
-    if (element.type === "password") {
-      attrs["value"] = element.value ? "*".repeat(element.value.length) : "";
-    } else {
-      attrs["value"] = element.value;
-    }
+    attrs["value"] = element.value;
```

Comment:
can you revert this change? it looks like something weired happened in syng PR 😂